lineman/app/services/mention_service.coffee

```diff
@@ -0,0 +1,7 @@
+angular.module('loomioApp').factory 'MentionService', ->
```

Comment:
I think the transforming is all client side at the moment?

server "I am a @user" ->
client display "I am a [@mention|user]" ->
client edit "I am a @user" ->
server "I am a @user"

If you give the branch a quick hoon I think it will be clear what's going
on.

On Wed, Oct 28, 2015 at 11:53 AM, Robert Guthrie <notifications@github.com>
wrote:

> In lineman/app/services/mention_service.coffee
> <https://github.com/loomio/loomio/pull/2645#discussion_r43234984>:
>
> > @@ -0,0 +1,7 @@
> > +angular.module('loomioApp').factory 'MentionService', ->
>
> I think I'm down with where you're going, particularly on not making a
> relation in the db.
>
> I'm happy to go with this, but still wish the server did not have to do a
> text transform, and that this transforming could be entirely client side.
> But.. I guess we can always change it later :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/loomio/loomio/pull/2645/files#r43234984>.
>